__tests__/app.test.js

```diff
@@ -346,3 +346,28 @@ describe("DELETE /api/comments/:comment_id", () => {
             });
     });
 });
+
+describe("GET /api/users", () => {
+    test("responds with a status code: 200", () => {
+        return request(app).get("/api/users").expect(200)
+    })
+    test("respond with an array of user objects each with the following properties: username, name, avatar_url", () => {
+        return request(app)
+            .get("/api/users")
+            .then(({ body }) => {
+                const { users } = body;
+                expect(Array.isArray(users)).toBe(true);
```

Comment:
No need to check for it being an array as are later using a forEach which proves it's an array 👍